### DISCUSSION TITLE
NextJs v10 not compatible with simplecrypto 2.2

### DISCUSSION THREAD

**Discussion by madhur77** [vercel/next.js#19073] [Help]
After upgrading to next 10 , simplecrypto is throwing error below, though key is passed from .env file, working fine in <10

this error is thrown 

**"SimpleCrypto object MUST BE initialised with a SECRET KEY."  for this '**

`code const simpleCrypto = new SimpleCrypto(process.env.secretKey);`

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

1. Use https://www.npmjs.com/package/simple-crypto-js with next 10 

## Expected behavior

Must read the key.

**Comment by kachkaev** [comment]
There does not seem to be anything specific to simple crypto. In essence, your code is:

```js
const myKey = process.env.secretKey; // ← the problem is here
const simpleCrypto = new SimpleCrypto(myKey);
```

See these Next.js docs:

- https://nextjs.org/docs/basic-features/environment-variables
- https://nextjs.org/docs/api-reference/next.config.js/runtime-configuration
